pandas-stubs/io/parsers/readers.pyi

```diff
@@ -57,7 +57,7 @@ def read_csv(
     skiprows: int | Sequence[int] | Callable[[int], bool] = ...,
     skipfooter: int = ...,
     nrows: int | None = ...,
-    na_values: Sequence[str] | Mapping[str, Sequence[str]] = ...,
+    na_values: Sequence[str] | Mapping[str, Sequence[str]] | None = ...,
```

Comment:
This change is not needed.  Only the defaults changed, and we don't put that in the stubs.  I will mark the issue as complete in #624

Reply:
Done